.github/workflows/build.yml

```diff
@@ -523,6 +553,35 @@ jobs:
           name: steamcmd logs
           path: build_output/
 
+  publish_pimax:
+    name: Publish Pimax Release
+    needs: [configuration, build]
+    runs-on: windows-latest
+    env:
+      PIMAX_APP_ID: ${{ secrets.PIMAX_APP_ID }}
+      PIMAX_USERNAME: ${{ secrets.PIMAX_USERNAME }}
+      PIMAX_PASSWORD: ${{ secrets.PIMAX_PASSWORD }}
+    if: |
+      github.event_name == 'push' &&
+      github.repository == 'icosa-gallery/open-brush' &&
+      (contains(github.ref, 'refs/tags/v'))
```

Comment:
Does Pimax have a concept of release channels?

Reply:
Not at this time, the tool only serves to upload the builds

Reply:
oof. Ok.